test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -7992,6 +7992,17 @@ def forward(self, x, y):
                       input_names=['input_x', 'input_y'],
                       dynamic_axes={'input_x': [0, 1, 2, 3], 'input_y': [0, 1, 2, 3]})
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_to_device(self):
+        class M(torch.nn.Module):
+            def forward(self, x, y):
+                return x.to(y.device)
```

Comment:
```suggestion
                return x.to(y.device), y
```

to keep y relevant, without above test input change.